airflow/contrib/operators/mysql_to_gcs.py

```diff
@@ -170,16 +176,20 @@ def _write_local_schema_file(self, cursor):
 
         self.log.info('Using schema for %s: %s', self.schema_filename, schema)
         tmp_schema_file_handle = NamedTemporaryFile(delete=True)
-        json.dump(schema, tmp_schema_file_handle)
```

Comment:
@jgao54, am I wrong or doesn't this break the MySQLToGCSOperator in Python 3? We're working on moving to Python 3 and this is causing issues for us.

Reply:
@mikeghen  we just encountered the same bug while attempting to run the operator on a table with binary fields, with the error that the byte type is not json serializable. I think the right approach is to use a custom JsonDecoder, that decode any byte values in the dictionary to string.